### ISSUE TITLE
Cannot run ffprobe and ffmpeg version

### ISSUE THREAD

**Issue by Behinder** [holzschu/a-Shell-commands#38]
Wasm ffmpeg gives Error: 
Wasm ffprobe gives `file ffprobe not found` same if I add filepath as argument
Both versions downloaded from this site.

**Comment by holzschu** [comment]
What about `wasm ffprobe.wasm`, or just `ffprobe`?

**Comment by Behinder** [comment]
After killing and relaunching a-shell ffprobe works. Wasm ffprobe.wasm shows file ffprobe.wasm not found so I assume ffprobe that launch successfully is the one built-in?

**Comment by holzschu** [comment]
I agree with your analysis (that ffprobe was the one built in). That leaves open the question: where is the ffprobe that you downloaded?

**Comment by Behinder** [comment]
~/Documents/bin

**Comment by holzschu** [comment]
Thanks. I assume you did check that it was present? (probably with `ls -l bin/ff*`) 

If it's in `~/Documents/bin`, then you need to call wasm with its actual location: `wasm bin/ffprobe.wam`. IIRC, if `ffprobe.wasm` is in `~/Documents/bin`, then it will be called by `ffprobe`, before the built-in version. `ffprobe.wam` should definitely be executed. You can check which version is called by looking at what is says at the top (if it says `enable-gpl`, then it's the wasm version).

**Comment by Behinder** [comment]
I put it myself there I thought it is desired location.
Ffprobe gives
`ffprobe version N-97747-gffae62d96c Copyright (c) 2007-2020 the FFmpeg developers
  built with clang version 13.0.0 (https://github.com/llvm/llvm-project bb31aec2cc85b6e2e85e687d086f85408b166c
42) `
Wasm bin/ffprobe.Wasm gives
`ffprobe version N-97747-gffae62d96c Copyright (c) 2007-2020 the FFmpeg developers
  built with clang version 13.0.0 (https://github.com/llvm/llvm-project bb31aec2cc85b6e2e85e687d086f85408b166c
42) `

so at least I see they are identical.
However I am not sure if shipped ffmpeg version is the latest one? I tried to download the ffmpeg source but apparently a-shell does not allow to configure make and make install.

the reason for checking this is as I currently have no Mac I find using ffmpeg on a-shell useful for recording television from HDHomerun. The recording mpg loooks fine if opened in FTPManager,(there is no ffplay tough so cannot play it natively) however such recordings converted to mpeg4 using this ffmpeg result in file that not displaying properly in Luma Fusion.
